### PR TITLE
Added new RawResponse to AuthenticateResponse

### PR DESCRIPTION
**Pull request by cviecco**
Adding RawResponse to AuthenticateResponse in order to allow JS API signResponse

### CODE REVIEW

File: u2ftoken/token.go [hunk 1]
```diff
@@ -123,6 +123,9 @@ type AuthenticateResponse struct {
 
 	// Signature is the P-256 ECDSA signature over the authentication data.
 	Signature []byte
+
+	// RawResponse is the raw response bytes as sent from the u2f token
```
Comment (titanous) [inline u2ftoken/token.go]: Please adjust this to say:

> RawResponse is the raw response bytes from the U2F token.

**Review by titanous — COMMENTED** [review]
This looks good! Can you also add a [DCO sign-off](https://flynn.io/docs/contributing#developer%E2%80%99s-certificate-of-origin)? Thanks!

**Comment by cviecco** [comment]
I think I signed the DCO... (git commit -s).. if not please ping me

**Comment by titanous** [comment]
@cviecco This PR looks good to go, but the DCO sign-off is still missing.

**Review by titanous — APPROVED** [review]

**Pull request merged**